synapse/models/inet.py

```diff
@@ -656,3 +672,25 @@ def getBaseModels():
         }
         name = 'inet'
         return ((name, modl), )
+
+        @modelrev('inet',201706121318)
+        def _revModl201706121318(self):
+
+            # account for the updated sub-property extraction for inet:url nodes
+            adds = []
+            rows = self.core.getRowsByProp('inet:url')
+
+            for i,p,v,t in rows:
+                norm,subs = self.core.getTypeNorm('inet:url', v)
+
+                fqdn = subs.get('fqdn')
+                if fqdn is not None:
+                    adds.append((i, 'inet:url:fqdn', fqdn, t))
+
+                ipv4 = subs.get('ipv4')
+                if ipv4 is not None:
+                    adds.append((i, 'inet:url:ipv4', ipv4, t))
+
+            if adds:
+                self.core.addRows(adds)
```

Comment:
We might also want to call formTufoByProp for the fqdn and ipv4 here so that we can still hook the events on migration.